runtime/runtime.go

```diff
@@ -1343,7 +1367,10 @@ func (r *interpreterRuntime) getBlockAtHeight(height uint64, runtimeInterface In
 
 func (r *interpreterRuntime) newGetCurrentBlockFunction(runtimeInterface Interface) interpreter.HostFunction {
 	return func(invocation interpreter.Invocation) trampoline.Trampoline {
-		height := r.getCurrentBlockHeight(runtimeInterface)
+		height, err := r.getCurrentBlockHeight(runtimeInterface)
```

Comment:
The call to the runtime interface should be wrapped, just like in all other cases:

```suggestion
		var height uint64
		var err error
		wrapPanic(func() {
			height, err = r.getCurrentBlockHeight(runtimeInterface)
		})
```